corehq/apps/userreports/tests/utils.py

```diff
@@ -130,3 +131,17 @@ def mock_sql_backend():
 def mock_datasource_config():
     return patch('corehq.apps.userreports.reports.data_source.get_datasource_config',
                  return_value=(get_sample_data_source(), None))
+
+
+def get_simple_xform():
+    xform = XFormBuilder()
+    xform.new_question('first_name', 'First Name', data_type='string')
+    xform.new_question('last_name', 'Last Name', data_type='string')
+    xform.new_question('children', 'Children', data_type='int')
+    xform.new_question('dob', 'Date of Birth', data_type='date')
+    xform.new_question('state', 'State', data_type='select', choices={
+        'MA': 'MA',
+        'MN': 'MN',
+        'VT': 'VT',
+    })
```

Comment:
79-line XForm represented in 10 lines of code. Nice compression ratio!

Reply:
Awesome!